sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/src/Internals/Profiling/ProfilingSessionEventSource.cs

```diff
@@ -0,0 +1,88 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+#nullable enable
+
+using System.Diagnostics.Tracing;
+
+namespace Azure.Monitor.OpenTelemetry
```

Comment:
Please use the namespace `Azure.Monitor.OpenTelemetry.AspNetCore.Internals.Profiling`, as we match the folder name to the namespace.

Reply:
Please follow the same pattern for any other files inside this folder.